backend/src/services/CampusHousingService.ts

```
@@ -0,0 +1,71 @@
+}
+
+const housingService = new CampusHousingService();
+const housingRouter = Router();
+
+housingRouter.get('/housing/sublets', (req: Request, res: Response) => {
+  const { roomType } = req.query;
+  const items = housingService.getSublets(roomType as string);
+  res.json({ success: true, data: items });
+});
+
+housingRouter.post('/housing/sublets', (req: Request, res: Response) => {
```

ESLint

[error] Replace `'/housing/sublets'` with `"/housing/sublets"`

(prettier/prettier)